### ISSUE TITLE
1.3.0 sensitive to require ordering

### ISSUE THREAD

**Issue by mhw** [seattlerb/minitest-focus#17]
I just updated a Rails project from `minitest-focus` 1.2.1 to 1.3.0 and immediately ran into an issue running the test suite:

```
<...>/vendor/bundle/ruby/2.6.0/gems/minitest-focus-1.3.0/lib/minitest/focus.rb:1:in `<top (required)>': uninitialized constant Minitest
Did you mean?  MiniMime (NameError)
        from <HOME>/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:88:in `require'
        from <HOME>/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:88:in `rescue in block in require'
        from <HOME>/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:65:in `block in require'
        from <HOME>/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:58:in `each'
        from <HOME>/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:58:in `require'
        from <HOME>/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler.rb:174:in `require'
        from <...>/config/application.rb:7:in `<top (required)>'
        from <...>/vendor/bundle/ruby/2.6.0/gems/spring-2.1.1/lib/spring/application.rb:92:in `require'
        from <...>/vendor/bundle/ruby/2.6.0/gems/spring-2.1.1/lib/spring/application.rb:92:in `preload'
        from <...>/vendor/bundle/ruby/2.6.0/gems/spring-2.1.1/lib/spring/application.rb:157:in `serve'
        from <...>/vendor/bundle/ruby/2.6.0/gems/spring-2.1.1/lib/spring/application.rb:145:in `block in run'
        from <...>/vendor/bundle/ruby/2.6.0/gems/spring-2.1.1/lib/spring/application.rb:139:in `loop'
        from <...>/vendor/bundle/ruby/2.6.0/gems/spring-2.1.1/lib/spring/application.rb:139:in `run'
        from <...>/vendor/bundle/ruby/2.6.0/gems/spring-2.1.1/lib/spring/application/boot.rb:19:in `<top (required)>'
        from <HOME>/.rbenv/versions/2.6.6/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        from <HOME>/.rbenv/versions/2.6.6/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        from -e:1:in `<main>'
```

A short-term fix that works for me is to disable `spring` by setting `DISABLE_SPRING` in the environment. (I'd also disabled `bootsnap` in the backtrace above, but it's disabling `spring` that fixes things for me.)

That said, `minitest-focus` does seem to now be sensitive to load order in a way that 1.2.1 wasn't. Specifically, [`focus.rb` in 1.2.1](https://github.com/seattlerb/minitest-focus/blob/v1.2.1/lib/minitest/focus.rb) required `minitest/test` before loading `minitest/focus5.rb`. In renaming the files to drop minitest 4 support this require has also been dropped - could it be added back in  to the top of `minitest/focus.rb`? I've tried this in my local bundle and it solves the problem, with or without `spring` and `bootsnap`.

**Comment by ryanseys** [comment]
Sent a PR here to fix: https://github.com/seattlerb/minitest-focus/pull/18

**Comment by ryanseys** [comment]
Should be fixed now in 1.3.1 @mhw

**Comment by mhw** [comment]
Yep, that fixes it. Cheers!
